bin/generate_test_data.rb

```diff
@@ -0,0 +1,170 @@
+require "set"
+require "pty"
+
+module Portus
+  # This class is the one responsible for generating new development data.
+  class TestData
+    # The number of users to be created.
+    NUSERS = 100
+
+    # Name of the only user that is not created with a random name.
+    SPECIAL_USER = "admin"
+
+    # Teams to be created.
+    TEAM_NAMES = [
+      "bigdata",
+      "containers",
+      "dev",
+      "hardware",
+      "networking",
+      "ops",
+      "qa",
+      "security",
+      "sitereliability",
+      "ui"
+    ]
+
+    # Create some random users + the admin.
+    def create_users!
+      logins = Set.new
+
+      while logins.size < NUSERS
+        first_name  = FFaker::Name.first_name
+        last_name   = FFaker::Name.last_name
+        username    = "#{first_name[0]}#{last_name}".downcase
+
+        next if username =~ /\W/ || username == SPECIAL_USER
+        next if logins.include? username
+
+        User.create!(
+          username: username,
+          password: "12341234",
+          email:    "#{username}@portusexample.com"
+        )
+        logins << username
+      end
+
+      User.create!(
+        username: SPECIAL_USER,
+        password: "12341234",
+        email:    "admin@portusexample.com",
+        admin:    true
+      )
+    end
+
+    # Create the needed teams. Each team will have between 5 to 20 members and
+    # between 2 to 5 namespaces.
+    def create_teams_and_namespaces!
+      users = User.all
+
+      TEAM_NAMES.each do |team|
+        members = users.sample(rand(5..20))
+        t = Team.create!(
+          name:         team,
+          owners:       members.pop(rand(1..4)),
+          contributors: members.pop(rand(0..members.size)),
+          viewers:      members
+        )
+
+        rand(2..5).times do
+          name = FFaker::Food.ingredient.downcase.gsub(/[^#{User::USERNAME_CHARS}]/, "")
+          name += rand(0..1000).to_s
+
+          Namespace.create!(
+            name:     name,
+            team:     t,
+            public:   rand(0..1) == 1,
+            registry: Registry.get
+          )
+        end
+      end
+    end
+
+    # Push images to the registry.
+    def push_images!
+      hostname = Registry.get.hostname
+
+      `docker logout #{hostname}`
+      `docker pull library/alpine:latest`
+
+      Rails.logger.info "Pushing images. This could take a while..."
```

Comment:
IIRC `rubocop` was complaining if I did this. Maybe I can disable that check for this :)